<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogFactoryManage</class>
 <widget class="QDialog" name="DialogFactoryManage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>870</width>
    <height>503</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>厂家信息维护</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>8</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>厂家索引：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="EdSearchFact">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>6</y>
     <width>481</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>33</y>
     <width>601</width>
     <height>461</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string>序号</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>厂家名称</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>简称</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Logo文件</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>网址</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>备注</string>
    </property>
   </column>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>614</x>
     <y>2</y>
     <width>251</width>
     <height>331</height>
    </rect>
   </property>
   <property name="title">
    <string>详细信息</string>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>60</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>生产厂家：</string>
    </property>
   </widget>
   <widget class="QLabel" name="LbFactory">
    <property name="geometry">
     <rect>
      <x>74</x>
      <y>20</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="LbForShort">
    <property name="geometry">
     <rect>
      <x>74</x>
      <y>50</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>60</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>简   称：</string>
    </property>
   </widget>
   <widget class="QLabel" name="LbLogo">
    <property name="geometry">
     <rect>
      <x>74</x>
      <y>80</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>60</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Logo文件：</string>
    </property>
   </widget>
   <widget class="QLabel" name="LbWebSite">
    <property name="geometry">
     <rect>
      <x>74</x>
      <y>110</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>60</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>网   址：</string>
    </property>
   </widget>
   <widget class="QLabel" name="LbRemark">
    <property name="geometry">
     <rect>
      <x>74</x>
      <y>140</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>60</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>备   注：</string>
    </property>
   </widget>
   <widget class="QLabel" name="LbPic">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>170</y>
      <width>171</width>
      <height>141</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="BtnSearch">
   <property name="geometry">
    <rect>
     <x>556</x>
     <y>7</y>
     <width>50</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>检 索</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>340</y>
     <width>131</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="BtnAdd">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>添   加</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="BtnInsert">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>插   入</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="BtnModify">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>修   改</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="BtnDelete">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>删   除</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="BtnClose">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>退   出</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>636</x>
     <y>357</y>
     <width>77</width>
     <height>120</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QPushButton" name="BtnSetTop">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>置   顶</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="BtnSetUp">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>向   上</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="BtnSetDown">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>向   下</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="BtnSetBottom">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>置   底</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
